packages/mobile/src/walletConnect/screens/Loading.tsx

```diff
@@ -0,0 +1,67 @@
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import { StackScreenProps } from '@react-navigation/stack'
+import React, { useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ActivityIndicator, StyleSheet, Text, View } from 'react-native'
+import { WalletConnectPairingOrigin } from 'src/analytics/types'
+import { Namespaces } from 'src/i18n'
+import { headerWithBackButton } from 'src/navigator/Headers'
+import { isScreenOnForeground, navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+
+const CONNECTION_TIMEOUT = 10_000
```

Comment:
Love this number notation!
There's an eslint rule we could add that can automatically format them this way.